style/ember/sample.js

```diff
@@ -16,3 +16,13 @@ test("checks the box", function() {
     ok(checkBox.prop("checked"), "box is checked");
   });
 });
+
+import Ember from 'ember';
+import DS from 'ember-data';
+
+const { computed } = Ember;
+const { attr, hasMany, belongsTo } = DS;
+
+export default DS.Model.extend({
+  ...
```

Comment:
What about only doing this on functions one level deep? i.e. not `Ember.Object` or `DS.Model` and not on `Ember.computed.inject`. Not sure how best to word this...